packages/server/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@sigstore/server",
```

Comment:
nit but should this be called something like `@sigstore/mock-server` to make it clearer it's only for testing purposes?